c++/src/kj/compat/tls.c++

```diff
@@ -599,10 +577,8 @@ public:
     //   So, we make some copies here.
     auto& tlsRef = tls;
     auto hostnameCopy = kj::str(hostname);
-    return inner->connectAuthenticated().then(
-        [&tlsRef, hostname = kj::mv(hostnameCopy)](kj::AuthenticatedStream stream) {
-      return tlsRef.wrapClient(kj::mv(stream), hostname);
-    });
+    auto stream = co_await inner->connectAuthenticated();
+    co_return co_await tlsRef.wrapClient(kj::mv(stream), hostname);
```

Comment:
Similarly, probably should be hostnameCopy instead of hostname here?

Reply:
not needed either